tests/integration/preparation.py

```diff
@@ -154,16 +130,14 @@ def create_set_of_nodes(first_node_id, nodes_number):
 
 
 def accelerate_skale_manager():
-    pr_key = os.environ.get('ETH_PRIVATE_KEY')
-    account = skale.web3.toChecksumAddress(private_key_to_address(pr_key))
+    skale = init_skale_with_base_wallet()
 
     reward_period = skale.validators_data.get_reward_period()
     delta_period = skale.validators_data.get_delta_period()
-    print(reward_period, delta_period)
+    print(f'New times for SM: {reward_period}, {delta_period}')
 
-    wallet = {'address': account, 'private_key': pr_key}
-    res = skale.constants.set_periods(TEST_EPOCH, TEST_DELTA, wallet)
-    receipt = await_receipt(skale.web3, res['tx'], retries=30, timeout=6)
+    res = skale.constants.set_periods(TEST_EPOCH, TEST_DELTA)
+    receipt = wait_receipt(skale.web3, res['tx'], retries=30, timeout=6)
     print(receipt)
     print("-------------------------")
```

Comment:
mb it's better to move this to constants?

Reply:
oh, thanks, forgot to delete it (just a temporary debug output)